examples/module1/lesson5/tanstack-query/components/Form.tsx

```diff
@@ -0,0 +1,72 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import axios from 'axios';
+import { FormEvent, useState } from 'react';
+import Input from './Input';
+
+export type Article = {
+  id: number;
+  author: string;
+  title: string;
+  content: string;
+};
+
+const formDefaultState: Article = {
+  id: 0,
+  author: '',
+  title: '',
+  content: '',
+};
+
+function Form() {
+  const [formState, setFormState] = useState(formDefaultState);
+  const queryClient = useQueryClient();
+  const mutation = useMutation({
+    mutationFn: (newArticle: Article) => {
+      return axios.post('/api/data/articles', newArticle);
+    },
+    onSuccess: () => {
+      queryClient.invalidateQueries();
```

Comment:
na pewno chcesz invalidować wszystkie query, które są w keszu po wykonaniu tej mutacji? nawet jeśli w apce ich nie ma dużo to jednak pokusiłbym się tutaj o przekazanie konkretnych kluczy